spec/features/user_show_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+
+RSpec.describe 'user_index', type: :feature do
+  user = User.first
+  before(:each) do
+    visit(user_path(User.first.id))
+  end
+
+  it 'shows the username of the the users' do
+    expect(page).to have_content('Bill')
+  end
```

Comment:
- [ ] Because you are implementing integration tests, I recommend creating a user so you'll be able to test the users_show page. Your tests fail because there are no users in your DB. For example, you are trying to test if the show page has the name 'Bill', but there is no user with the name 'Bill'.

<img width="475" alt="image" src="https://user-images.githubusercontent.com/77190203/187891313-3813b96d-80f7-452a-91bc-ef74af64afc5.png">

Reply:
Done ✔